packages/bpk-docs/src/pages/ChipsPage/ChipsPage.js

```diff
@@ -113,7 +113,6 @@ const ChipsPage = () => (
     ]}
     components={components}
     readme={chipReadme}
-    sassdocId="cards"
```

Comment:
is this change intentional? if so ignore this comment 😄

Reply:
Aye! This is the Chips page - that `sassdocId` is pointing to cards!! copypasta error.